flocker/node/agents/blockdevice.py

```diff
@@ -1299,6 +1318,7 @@ class RawState(PClass):
     The raw state of a node.
 
     :param unicode compute_instance_id: The identifier for this node.
+    :param unicode live_instances: The identifiers of live nodes.
```

Comment:
nit:
:type live_instances: ``pset`` of unicode node identifiers, rather than just `unicode` type.